tests/commands/bulk_test.py

```diff
@@ -0,0 +1,80 @@
+# pylint: disable=no-self-use,invalid-name
+import argparse
+import json
+import os
+import tempfile
+from unittest import TestCase
+
+from allennlp.__main__ import main
+from allennlp.commands.bulk import add_bulk_subparser, bulk
+
+
+class TestMain(TestCase):
+
+    def test_add_bulk_subparser(self):
+        parser = argparse.ArgumentParser(description="Testing")
+        subparsers = parser.add_subparsers(title='Commands', metavar='')
+        add_bulk_subparser(subparsers)
+
+        raw_args = ["bulk",     # command
+                    "reverser", # model
+                    "infile",   # input_file
+                    "--output-file", "outfile",
+                    "--print"]
+
+        args = parser.parse_args(raw_args)
+
+        assert args.func == bulk
+        assert args.model == "reverser"
+        assert args.input_file == "infile"
+        assert args.output_file == "outfile"
+        assert args.print
```

Comment:
`print` is a boolean flag, pylint doesn't like it if you do `assert args.print == True`